content/rest-api/filter.md

```diff
@@ -1013,15 +1013,43 @@ As a result you will receive the following answer:
 ```
 
 ## Filter attributes
+
+When you request specific attributes, you can use filters to select only the ones you want. 
+
+### By attribute codes
 ::: availability versions=Serenity editions=CE,EE
 
-When you need to request attributes, you can use a filter to select only the ones with specific codes.
+You can filter the attributes by their code.
+
+Below you will find the operator allowed to filter on this property, as well as the corresponding type of value required in the `search` query parameter.
+
+| Operator | Allowed value type | Filter description |
+| ----------------- | -------------- | ------------------ |
+| `IN` | an array of existing attribute codes | Only returns attributes that are in the list |
+
+#### Example
+```
+/api/rest/v1/attributes?search={"code":[{"operator":"IN","value":["code1","code2"]}]}
+```
+
+### By update date
+::: availability versions=Serenity editions=CE,EE
+
+You can filter the attributes by their update date.
+
+Below you will find the operator allowed to filter on this property, as well as the corresponding type of value required in the `search` query parameter.
 
 :::info
-Filtering attributes currently only works with attribute codes and the "IN" operator.
+Please note that dates must follow the [ISO 8601 standard](https://en.wikipedia.org/wiki/ISO_8601).
 :::
 
+| Operator | Allowed value type | Filter description |
+| ----------------- | -------------- | ------------------ |
+| `>` | datetime <br> _Format: ISO 8601_ | Only returns attributes that were <br> updated after the given day and hour |
```

Comment:
if I update brand attribute option => the request doesn’t return the brand attribute as updated. But if I update an attribute property => the request returns the brand attribute as updated. Maybe it's worth adding it to the documentation.